### PR TITLE
New package: Octol1ttle.ElyPrismLauncher version 8.4

### PR DESCRIPTION
**Pull request by znarfm**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163430)

### CODE REVIEW

**Comment by znarfm** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Programs\PrismLauncher\prismlauncher.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\PrismLauncher\prismlauncher_filelink.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\PrismLauncher\prismlauncher_updater.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 886.)

File: manifests/o/Octol1ttle/ElyPrismLauncher/8.4/Octol1ttle.ElyPrismLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Octol1ttle.ElyPrismLauncher
+PackageVersion: "8.4"
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/o/Octol1ttle/ElyPrismLauncher/8.4/Octol1ttle.ElyPrismLauncher.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @znarfm,

This package appears to have a dependency. 

image

Edit: Installed above dependency - is Java also a dependency?
image

**Comment by stephengillie** [comment]
Verify with above dependencies:
image
image

Just to ask - we already have a package named `PrismLauncher.PrismLauncher` which is also at version `8.4` - could this be the same package?

**Comment by znarfm** [comment]
> could this be the same package?

This package is actually a fork of Prism Launcher. It includes additional features not available to the original package, such as the ability to log in with an Ely.by account.

**Review by stephengillie — APPROVED** [review]

**Comment by mdanish-kh** [comment]
[Policy] Needs-Manual-Merge

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**